eng/pipelines/templates/stages/platform-matrix.json

```diff
@@ -20,7 +20,8 @@
     "NodeTestVersion": [
       "16.x",
```

Comment:
Could we also drop 16 because it has reached end of life and no longer officially supported?
```suggestion
```

Reply:
yes it's dropped with Jeff's pr https://github.com/Azure/azure-sdk-for-js/pull/27553